src/providers/context.tsx

```diff
@@ -70,7 +70,7 @@ export interface ApplicationContextValue extends State {
   clearContext: () => Promise<void>;
   showActivityIndicator: (message?: string) => void;
   hideActivityIndicator: () => void;
-  loadAppData: () => Promise<void>;
+  loadAppData: () => Promise<Error | true>;
```

Comment:
This PR doesn't change what this function does, just updates the types to accurately reflect what it does